meta/admin.py

```diff
@@ -3,13 +3,22 @@
 from meta.models import *
 from modeltranslation.admin import TranslationAdmin
 
+class CuradoriaAdmin(admin.ModelAdmin):
```

Comment:
Wouldn't `CuratorshipAdmin` be better?

Reply:
Yes, we will change it in the next PR. This PR is just for the other developers to have access to the code